messages/pt-BR.json

```diff
@@ -1,4 +1,5 @@
 {
   "editor.product-search.title": "Contexto de busca de produtos",
-  "editor.product-search.maxItemsPerPage": "Número máximo de elementos por página"
-}
\ No newline at end of file
+  "editor.product-search.maxItemsPerPage": "Número máximo de elementos por página",
+  "store.network-status.offline": "Sem conexão"
```

Comment:
I think this message is not good.  _"You are offline. Please check your connection"_ or something like this. @oldufre can help with this.